Services/User/classes/class.ilRoleAssignmentTableGUI.php

```diff
@@ -109,9 +109,30 @@ protected function fillRow(array $a_set): void // Missing array type.
                 $this->tpl->setVariable('VAL_CHECKED', 'checked="checked"');
             }
         }
-        $this->tpl->setVariable("ROLE", $a_set["role"]);
-        $this->tpl->setVariable("DESCRIPTION", $a_set["description"]);
-        $this->tpl->setVariable("CONTEXT", $a_set["context"]);
+        $this->tpl->setVariable(
+            'ROLE',
+            $this->renderer->render(
+                $this->factory->link()->standard(
+                    ilObjRole::_getTranslation($a_set['title']),
+                    $this->ctrl->getLinkTargetByClass(ilObjRoleGUI::class, 'perm')
+                )
+            )
+        );
```

Comment:
I think we are missing the two parameters (`$obj_id` of the role and `$ref_id` of the role folder - see below) here when setting the link.

Reply:
IMO we already have the `ref_id`, because the role folder is the parent.